CODEOWNERS

```diff
@@ -1,5 +1,6 @@
 # These are the default owners for the whole content of the repository. The default owners are automatically added as reviewers when you open a pull request, unless different owners are specified in the file.
 * @tobiscr @pbochynski @crabtree @varbanv @rJankowski93 @suleymanakbas91
+/cmd @kyma-incubator/wookiee
```

Comment:
Hope its enough to be code owner just for this folder.. please feel free to add further codeowner ships if you see it's not sufficient for you guys